lib/win32.js

```diff
@@ -29,6 +29,21 @@ const handleOptions = opts => {
 };
 
 /**
+ * Sets the value of the `hidden` attribute on dependency paths
+ * @param deps Object
+ * @param hidden Boolean
+ * @returns {Promise}
+ */
+const setHiddenAttr = (deps, hidden) =>
+  Promise.all(
+    deps.map(dep =>
+      fswin.setAttributesSync(dep.path, {IS_HIDDEN: hidden}) ?
+        Promise.resolve(Object.assign(dep, {hidden})) :
+        Promise.reject(new Error(`Failed to modify dir attr: ${dep.path}`))
```

Comment:
Yeah, the entire method here could actually be fully synchronous. I kept it pseudo-`async` though because it's consistent with the other methods and it's `then`able. Also it might be true `async` in the future if `fswin` gets fixed.